platform_disk_api/project_namespace_migration_job.py

```diff
@@ -0,0 +1,356 @@
+) -> None:
+    logger.info("waiting for pvc deletion: %s", pvc_name)
+    url = client._generate_pvc_url(namespace=namespace, pvc_name=pvc_name)
+    async for _ in _waiter():
+        try:
+            await client.get(url)
+        except ResourceNotFound:
+            return
+
+
+async def remove_claim_ref(
+        client: KubeClient,
+        pv_name: str,
+) -> None:
+    logger.info("removing claim ref: %s", pv_name)
+    url = f"{client.api_v1_url}/persistentvolumes/{pv_name}"
+    await client.patch(
+        url,
+        headers={"Content-Type": "application/merge-patch+json"},
+        json={
+            "spec": {
+                "claimRef": None
+            }
+        }
+    )
+    logger.info("claim ref removed: %s", pv_name)
+
+
+async def wait_claim_ref_set(
+        client: KubeClient,
+        pv_name: str,
+        pvc_name: str,
+) -> None:
+    logger.info("Waiting for claim ref to be set: %s; pv=%s", pvc_name, pv_name)
+    url = f"{client.api_v1_url}/persistentvolumes/{pv_name}"
+    async for _ in _waiter():
+        pv = await client.get(url)
+        if (pv["spec"].get("claimRef", {}) or {}).get("name") == pvc_name:
+            return
+
+
+async def create_pvc(
+        client: KubeClient,
+        pvc_name: str,
+        old_pvc: dict[str, Any],
+        namespace: str,
+        org_name: str,
+        project_name: str,
+        pv_name: Optional[str] = None,
+) -> None:
+    logger.info(
+        "creating a new PVC: %s; namespace=%s; org_name=%s; project_name=%s; pv=%s",
+        pvc_name, namespace, org_name, project_name, pv_name
+    )
+    old_spec = old_pvc["spec"]
+    old_metadata = old_pvc["metadata"]
+
+    annotations = {}
+
+    for annotation_key in (
+        DISK_API_NAME_ANNOTATION,
+        DISK_API_CREATED_AT_ANNOTATION,
+        DISK_API_LAST_USAGE_ANNOTATION,
+        DISK_API_LIFE_SPAN_ANNOTATION,
+        DISK_API_USED_BYTES_ANNOTATION,
+    ):
+        if annotation_key in old_metadata["annotations"]:
+            annotations[annotation_key] = old_metadata["annotations"][annotation_key]
+
+    spec = {
+        "accessModes": old_spec.get("accessModes", []),
+        "resources": old_spec.get("resources", {}),
+        "storageClassName": old_spec["storageClassName"],
+        "volumeMode": old_spec.get("volumeMode", "Filesystem")
+    }
+
+    if pv_name:
+        spec["volumeName"] = pv_name
+
+    new_pvc_body = {
+        "apiVersion": "v1",
+        "kind": "PersistentVolumeClaim",
+        "metadata": {
+            "name": pvc_name,
+            "namespace": namespace,
+            "uid": old_metadata["uid"],  # keep old UID so PV can claim the new PVC
+            "labels": {
+                APOLO_ORG_LABEL: org_name,
+                APOLO_PROJECT_LABEL: project_name,
+                DISK_API_ORG_LABEL: org_name,
+                DISK_API_MARK_LABEL: "true",
+                PROJECT_LABEL: project_name,
+                USER_LABEL: old_metadata["labels"][USER_LABEL],
```

Comment:
Can you add also these labels but with `apolo.us` in key? let's eventually migrate to new labels. Same for annotations